pkg/storage/truncate.go

```diff
@@ -22,6 +22,7 @@ func (s *Store) TruncateAllTablesDANGEROUS(logger *zap.Logger) error {
     plan_it_tools,
     plan_favorite,
     plan_cr_tdl,
+	operational_solution_subtask,
```

Comment:
Sorry I think I forgot to submit this comment last time. There is an error message about this when you run `scripts/dev test:go` unit test, caused because this needs to be replicated there. Could you please add it there as well? 

```
ERROR:  cannot truncate a table referenced in a foreign key constraint
DETAIL:  Table "operational_solution_subtask" references "operational_solution".
HINT:  Truncate table "operational_solution_subtask" at the same time, or use TRUNCATE ... CASCADE.
```